### PR TITLE
add mufu.sqrt for GM200+

### PR DESCRIPTION
**Pull request by karolherbst**
found today:
MUFU.SQRT R22, R22;                                                 /* 0x5080000000871616 */

### CODE REVIEW

File: envydis/gm107.c [hunk 1]
```diff
@@ -1668,6 +1668,7 @@ static struct insn tab5080_0[] = {
 	{ 0x0000000000500000ull, 0x0000000000f00000ull, N("rsq") },
 	{ 0x0000000000600000ull, 0x0000000000f00000ull, N("rcp64h") },
 	{ 0x0000000000700000ull, 0x0000000000f00000ull, N("rsq64h") },
+	{ 0x0000000000800000ull, 0x0000000000f00000ull, N("sqrt"), .fmask = F_SM52 },
```
[the remark this answers is not in the document]
Comment (karolherbst) [reply envydis/gm107.c]: I already fixed that by cleaning up the variant stuff at the bottom

**Comment by imirkin** [comment]
And by "cleaning up" you mean "breaking"? There is some SM_50-only stuff
and some SM_60-only stuff.

**Comment by karolherbst** [comment]
ohhh, didn't saw that one. Okay

**Comment by karolherbst** [comment]
@imirkin so that "| F_SM60" didn't work out either. I changed the patch so that the SM50 stuff isn't added to SM60, butthe SM52 one is.

Maybe there is a cleaner solution?

**Pull request merged**